locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nOBSERVACI&Oacute;: totes les vostres notes seran enviades a la paperera.
```

Comment:
I have seen that behavior before where sometimes the html is processed properly and sometimes it shows the literal strings, so I understand what you're saying. Unfortunately I'm just watching this app for contributions -- I'm not the developer so I'm not sure where the root of the problem is. 

Regarding how to proceed, the strings do display correctly for me when I use "Paràmetres" and "Observació" in the properties file directly. If you want to take that shortcut, it will probably work and I can push out an update. 

![screen shot 2015-12-22 at 12 31 36 pm](https://cloud.githubusercontent.com/assets/2423092/11965199/fee61dc0-a8a7-11e5-8c69-50dd648b30a0.png)